app/controllers/posts_controller.rb

```diff
@@ -6,5 +6,26 @@ def index
 
   def show
     @post = Post.find(params[:id])
+    @author = User.find(@post.author_id)
+    @new_post = current_user.posts.build
+  end
+
+  def new
+    @post = Post.new
+  end
```

Comment:
- [ ] Great job so far @bohaz Just as the previous reviewer mentioned, I am also unable to create a post, and even though the action for creating a post has been included, there isn't a corresponding view file or form present that can enable users to create a post so kindly fix this :wink: 
- Error Message
![image](https://github.com/bohaz/BlogApp/assets/102699958/2ca7e4f5-656b-441a-b2fd-d06aec0a7ad0)


- [ ] Kindly ensure that you do the same for the `comments creation` as well so that there is a form that makes it possible for users to create comments for a post as well :wink:

Reply:
Hi @NahnahAJ 
I hope you are doing well.
I notice that you are looking for the forms in a new page that doesn't exist in my application. Please click on a post, there are two forms for creating a new post and also a new comment. If it's mandatory to create a new page for the forms please let me know.

Reply:
![image](https://github.com/bohaz/BlogApp/assets/127757182/aa9e5504-23bf-4b73-b724-5a706405f68a)
![image](https://github.com/bohaz/BlogApp/assets/127757182/764c8856-5c6a-440d-9a72-158417e0e581)
![image](https://github.com/bohaz/BlogApp/assets/127757182/4e98c504-9208-4987-a081-aac99e6ae215)